packages/react/generators.json

```diff
@@ -95,6 +95,13 @@
       "schema": "./src/generators/setup-tailwind/schema.json",
       "description": "Set up Tailwind configuration for a project.",
       "hidden": false
+    },
+
+    "setup-ssr": {
+      "factory": "./src/generators/setup-ssr/setup-ssr#setupSsrSchematic",
+      "schema": "./src/generators/setup-ssr/schema.json",
+      "description": "Set up SSR configuration for a project.",
```

Comment:
I know it's a little verbose, but can we mention `react project` to prevent any ambiguity?